modules/ServerAPI/src/server/routes/ServerUtilityFunctions.coffee

```diff
@@ -836,10 +838,21 @@ class Thing extends Backbone.Model
 		if @lsProperties.defaultLabels?
 			for dLabel in @lsProperties.defaultLabels
 				newLabel = @get('lsLabels').getOrCreateLabelByTypeAndKind dLabel.type, dLabel.kind
+				@listenTo newLabel, 'createNewLabel', @createNewLabel
 				@set dLabel.key, newLabel
 				#			if newLabel.get('preferred') is undefined
 				newLabel.set preferred: dLabel.preferred
 
+	createNewLabel: (lKind, newText) =>
```

Comment:
your trigger includes the key as the third argument, but you ignore it in this implementation. I think you should set the newLabel key. See Thing.coffee around line 222